src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainSpecBasedSpecProvider.cs

```diff
@@ -259,11 +259,27 @@ private void SetMaxAndTargetBlobCount(ChainSpec chainSpec, ReleaseSpec spec, ulo
             {
                 spec.TargetBlobCount = blobCount.Target;
                 spec.MaxBlobCount = blobCount.Max;
+                if (blobCount.BaseFeeUpdateFraction == 0)
+                {
+                    if (fork.Name == "Cancun")
+                    {
+                        spec.BlobBaseFeeUpdateFraction = Eip4844Constants.BlobGasPriceUpdateFractionCancun;
+                    } else if (fork.Name == "Prague")
+                    {
+                        spec.BlobBaseFeeUpdateFraction = Eip4844Constants.BlobGasPriceUpdateFractionPrague;
+                    }
+                }
+                else
+                {
+                    spec.BlobBaseFeeUpdateFraction = blobCount.BaseFeeUpdateFraction;
+                }
```

Comment:
this is a temp things, untill [this](https://github.com/ethereum/EIPs/pull/9240/files) gets merged